docs/source/WritingTests/WritingSimpleTests.rst

```diff
@@ -167,6 +162,45 @@ to pick up a living guest, connect to it via ssh, and return its uptime.
            logging.info("Guest uptime result is: %s", uptime)
            session.close()
 
+#. Note that all failures that might happen here are implicitly handled
+   by the methods called. If a test went from its beginning to its end
+   without unhandled exceptions, avocado assumes the test automatically
+   as **PASS**, *no need to mark a test as explicitly passed*. If you
+   have explicit points of failure, for more complex tests, you might
+   want to mark it explicitly. ``test.cancel`` makes the test **CANCEL**
+   (for avocado 36lts version, as this method not backport yet, you may
+   want to use ``test.skip`` to make the test **SKIP** to achieve the
+   similar purpose), ``test.error`` makes the test **Error**, and
+   ``test.fail`` makes the test **Fail**. Note, people should never
+   import ``exceptions`` from ``avocado.core`` to raise them in test
```

Comment:
But in recent Avocado versions some of the exceptions are part of public API available directly in `avocado` and these are fine to be used. (and soon they will be part of the new LTS version)